app/controllers/volunteer_applications_controller.rb

```diff
@@ -7,6 +7,8 @@ def create
     @volunteer_application =
       VolunteerApplication.new(volunteer_application_params)
     if @volunteer_application.save
+      NewVolunteerApplicationMailer.volunteer_notification(
```

Comment:
So I have changed my code as suggested as it does read better. The thing that threw me is the syntax highlighting changes. So NewVolunteerApplicationMailer no longer highlights as a class. I have submitted a form and the mailer went out so it works, but it deterred me initially.